TradeItIosTicketSDK2/TradeItIosTicketSDK2.h

```diff
@@ -4,5 +4,9 @@
 #import "TradeItIosEmsApiLib.h"
 #import "YFTopGradientView.h"
 #import "YFLinearGradientView.h"
+#import "SerializedLinkedBroker.h"
```

Comment:
Maybe rename these to `CachedLinkedBroker` etc to make it clearer what these are used for.